### PR TITLE
Feature/footer

### PR DESCRIPTION
**Pull request by danielcfuentes**
Added the footer to all the pages and added some css:

image

### CODE REVIEW

**Review by janicetsai1 — COMMENTED** [review]
i notice that the 3 PRs you have open right now all have a lot of duplicate code. i  assumed this is because all are branched from main. can you instead make sure that if there are dependencies, to branch off of the branch it has dependencies on (like we discussed in our chat offline)? it makes it easier to review since there's a lot of code in this PR that would be redundant to review and not relevant to the footer. it's fine if you leave as is for now but just for next time!

**Review by janicetsai1 — APPROVED** [review]

File: frontend-runitup/src/Components/Authentication/LoginPage.jsx [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+  };
+
+  const handleChangePassword = (e) => {
+    setPassword(e.target.value);
+  };
+
+  const handleLogin = (e) => {
+    e.preventDefault();
+
+    fetch(`${import.meta.env.VITE_BACKEND_ADDRESS}/login`, {
```
Comment (jayeshdoolani) [inline frontend-runitup/src/Components/Authentication/LoginPage.jsx:22]: I like this v much!

File: frontend-runitup/src/Components/Authentication/LoginPage.jsx [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+          throw new Error("Failed to login");
+        }
+      })
+      .then((data) => {
+        onLogin({ name: username }, data.accessToken, data.refreshToken);
+        navigate("/feed");
+      })
+      .catch((error) => {
+        console.error("Error:", error);
+      });
```
Comment (jayeshdoolani) [inline frontend-runitup/src/Components/Authentication/LoginPage.jsx]: Ok for development, but make sure this doesn't go into the prod. Logging error into console might reveal private info about the user or website

File: frontend-runitup/src/Components/Authentication/SignUp.jsx [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+  };
+
+  const handleCreate = (e) => {
+    e.preventDefault();
+
+    fetch(`${import.meta.env.VITE_BACKEND_ADDRESS}/create`, {
+      method: "POST",
+      headers: {
+        "Content-Type": "application/json",
+      },
```
Comment (jayeshdoolani) [inline frontend-runitup/src/Components/Authentication/SignUp.jsx]: `headers` is repeated between signup and login. Make sure to modularize this later

**Review by jayeshdoolani — APPROVED** [review]
Highlighted some minor nits, no need to address them in this PR but just for future reference

File: frontend-runitup/src/Components/Authentication/LoginPage.jsx (hunk 2 above)
Comment (danielcfuentes) [reply frontend-runitup/src/Components/Authentication/LoginPage.jsx]: Thank you for pointing that out. I will go and fix that and make sure to remove those console messages when going into prod!

File: frontend-runitup/src/Components/Authentication/SignUp.jsx (hunk 3 above)
Comment (danielcfuentes) [reply frontend-runitup/src/Components/Authentication/SignUp.jsx]: I created a file in the untils folder that contains the headers. Then pass that into the sign up and login pages.

**Pull request merged**